nifi-nar-bundles/nifi-update-attribute-bundle/nifi-update-attribute-processor/src/main/java/org/apache/nifi/processors/attributes/UpdateAttribute.java

```diff
@@ -143,9 +144,10 @@ public ValidationResult validate(String subject, String input, ValidationContext
         }
     };
 
+    public static final String DELETE_ATTRIBUTES_EXPRESSION_NAME = "Delete Attributes Expression";
     // static properties
     public static final PropertyDescriptor DELETE_ATTRIBUTES = new PropertyDescriptor.Builder()
-            .name("Delete Attributes Expression")
+            .name(DELETE_ATTRIBUTES_EXPRESSION_NAME)
```

Comment:
Since you're touching this anyway, perhaps add a `.displayName(DELETE_ATTRIBUTES_EXPRESSION_NAME)` as well (if it doesn't affect existing flows, that is)